### PR TITLE
Add post "GrabShare at the Intelligent Transportation Engineering Conference"

### PR DESCRIPTION
**Pull request by paradite**
screenshot from 2017-12-13 19-39-05

### CODE REVIEW

File: _posts/2017-12-13-grabshare-at-the-intelligent-transportation-engineering-conference.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+date: 2017-12-13 06:00:00
+authors: [dominic-widdows]
+categories: [Data Science]
+tags: [Data Science, GrabShare]
+comments: true
+cover_photo: /img/grabshare-at-the-intelligent-transportation-engineering-conference/cover.jpg
+excerpt: "We’re excited to share the publication of our paper GrabShare: The Construction of a Realtime Ridesharing Service, which was Grab’s contribution to the Intelligent Transportation Engineering Conference in Singapore last month."
+---
+
+We’re excited to share the publication of our paper [GrabShare: The Construction of a Realtime Ridesharing Service](http://ieeexplore.ieee.org/document/8056896/), which was Grab’s contribution to the [Intelligent Transportation Engineering Conference](http://icite.org) in Singapore last month.
```
Comment (yangshun) [inline _posts/2017-12-13-grabshare-at-the-intelligent-transportation-engineering-conference.md]: I would change all the `’` to `'`.

File: _posts/2017-12-13-grabshare-at-the-intelligent-transportation-engineering-conference.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+We’ve come this far for GrabShare. It’s been a rewarding journey, and we will continue to iterate and innovate. According to our records and estimates, in the past month alone GrabShare saved over 4.5 million km in driving distance by using one car instead of two for thousands of shared journeys. In addition, the service has  reduced congestion and pollution including CO2 and other emissions – by about as much as 1,000 flights from Singapore to Beijing, or about as much CO2 as what 5 square kilometers of forest absorbs in a month. (As far as we can tell from researching on the web – we’re tree enthusiasts, not tree scientists!) And the travel cost savings have been attracting new passengers to the platform – within just two weeks in August, more than 100,000 new users took GrabShare rides.
```
Comment (yangshun) [inline _posts/2017-12-13-grabshare-at-the-intelligent-transportation-engineering-conference.md]: `has  reduced` has double spaces.

Comment (yangshun) [reply _posts/2017-12-13-grabshare-at-the-intelligent-transportation-engineering-conference.md]: Would be good to make 2 in CO2 `<sub>`.

**Comment by yangshun** [comment]
4 mins too late 😭

**Comment by paradite** [comment]
ah i will make changes you proposed anyway

**Comment by paradite** [comment]
fixed via https://github.com/grab/engineering-blog/commit/70fa6f973b8fb37927b3a4d68fe32d96db544755

**Pull request merged**